src/components/Registries/RegistriesCompanies/index.js

```diff
@@ -1,88 +1,132 @@
-import React, {useEffect, useState} from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import {NavLink, useParams} from 'react-router-dom';
-import {blockchainSelectors, registriesSelectors} from "../../../redux/selectors";
-import Card from "../../Card";
-import {registriesActions} from "../../../redux/actions";
-import Button from "../../Button/Button";
+import { NavLink } from 'react-router-dom';
+import { blockchainSelectors, registriesSelectors } from '../../../redux/selectors';
+import Card from '../../Card';
+import { registriesActions } from '../../../redux/actions';
+import Button from '../../Button/Button';
 import styles from './styles.module.scss';
-import {buildRegistryForm, renderRegistryItemDetails} from "../../../utils/registryFormBuilder";
-import DeleteCompanyModal from "../../Modals/DeleteCompanyModal";
-import {newCompanyDataObject} from "../../../utils/defaultData";
+import { RenderRegistryItemDetails } from '../../../utils/registryFormBuilder';
+import DeleteCompanyModal from '../../Modals/DeleteCompanyModal';
 import router from '../../../router';
 
-const RegistriesCompanies = () => {
-  const [expandedDetailsForCompany, setExpandedDetailsForCompany] = useState(null)
-  const [isDeleteCompanyModalOpen, setIsDeleteCompanyModalOpen] = useState(false)
+function RegistriesCompanies() {
+  const [expandedDetailsForCompany, setExpandedDetailsForCompany] = useState(null);
+  const [isDeleteCompanyModalOpen, setIsDeleteCompanyModalOpen] = useState(false);
   const dispatch = useDispatch();
   const userWalletAddress = useSelector(blockchainSelectors.userWalletAddressSelector);
   useEffect(() => {
-    dispatch(registriesActions.getOfficialUserRegistryEntries.call(userWalletAddress))
-  }, [dispatch, registriesActions])
+    dispatch(registriesActions.getOfficialUserRegistryEntries.call(userWalletAddress));
+  }, [dispatch, registriesActions]);
```

Comment:
```suggestion
  }, [dispatch, registriesActions]);
```
`registriesActions` is an imported module, it won't change and I believe eslint knows this and won't complain
`userWalletAddress` though must be in dependencies

(yea, I know it was already like that, but let's fix this as you've already touched it :P)